server/config/config.go

```diff
@@ -8,25 +8,22 @@ type StoredConfig struct {
 	OAuth2Authority    string
 	OAuth2ClientID     string
 	OAuth2ClientSecret string
-
+	bot.Config
 	EnableStatusSync   bool
 	EnableDailySummary bool
-
-	bot.Config
 }
 
 // Config represents the the metadata handed to all request runners (command,
 // http).
 type Config struct {
-	StoredConfig
-
+	PluginID               string
 	BuildDate              string
 	BuildHash              string
 	BuildHashShort         string
 	MattermostSiteHostname string
 	MattermostSiteURL      string
-	PluginID               string
 	PluginURL              string
 	PluginURLPath          string
 	PluginVersion          string
+	StoredConfig
```

Comment:
Unrelated change?

Reply:
@hanzei The branch was not in sync with the master.Now it is fixed